core/src/main/res/values/arrays.xml

```diff
@@ -269,6 +269,25 @@
         <item>is_favorite</item>
     </string-array>
 
+    <string-array name="episode_filter_selection_text1">
```

Comment:
I think those are no longer needed. Also, the array above (old dialog names/values) is no longer needed, as well.